src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerProvider.cs

```diff
@@ -21,10 +25,20 @@ public ApplicationInsightsLoggerProvider(TelemetryClient telemetryClient, Func<s
             this.filter = filter;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationInsightsLoggerProvider"/> class.
+        /// </summary>
+        public ApplicationInsightsLoggerProvider(TelemetryClient telemetryClient, IOptions<ApplicationInsightsLoggerOptions> options)
+        {
+            this.telemetryClient = telemetryClient;
+            this.filter = trueFilter;
```

Comment:
Why use a true filter here?

Reply:
I followed pattern in [ConsoleWriter](https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging.Console/ConsoleLoggerProvider.cs), but it is not needed with new approach.